### PR TITLE
Updates to Analytics

### PR DESCRIPTION
**Pull request by mattjstar**
- Use Segment (for Google Analytics and Mixpanel...and potentially others)
- Add Click Tracking with analytics.js
- Add link to careers page:

screen shot 2016-01-10 at 11 40 50 am

### CODE REVIEW

**Comment by mattjstar** [comment]
@i8ramin @OnFreund wanted to get your feedback before merging this stuff in.

Specifically whether you think that /careers link ("Join Our Team") is appropriate?

File: _includes/navigation.html [hunk 1]
```diff
@@ -2,3 +2,10 @@
 <a href="{{ site.baseurl }}/about/">About</a>
 <a href="{{ site.baseurl }}/contact/">Contact</a>
 -->
+
+<a
```
Comment (OnFreund) [inline _includes/navigation.html:6]: Can you add a screenshot?

**Comment by mattjstar** [comment]
screen shot 2016-01-10 at 1 42 03 pm

**Comment by OnFreund** [comment]
:+1:

**Pull request merged**